tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -61,8 +62,9 @@
 #define BTP_SERVICE_ID_VCP	20
 #define BTP_SERVICE_ID_CAS	21
 #define BTP_SERVICE_ID_MCP	22
+#define BTP_SERVICE_ID_GMCS	23
 
-#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_MCP
+#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_GMCS
```

Comment:
Are we going to have both `MCS` and `GMCS` here?

Reply:
Added BTP_SERVICE_ID_MCS (24) for future use.